torch/ao/quantization/fx/convert.py

```diff
@@ -686,7 +708,8 @@ def replace_observer_with_quantize_dequantize_node(
 
     # this is a temporary hack for custom module, we may want to implement
     # this properly after the custom module class design is finalized
-    def replace_observer_with_dequantize_node(node: Node, graph: Graph):
+    # TODO: Use this only for replacing DeQuantStubs; currently it's also used for replacing observers
+    def replace_with_dequantize_node(node: Node, graph: Graph):
```

Comment:
nit: `replace_observer_or_dequant_stub_with_dequantize_node` would be a more descriptive name